modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/instance-list-page/modal/instance-details/InstanceDetailsModalBody.es.js

```diff
@@ -33,10 +33,26 @@ function Body({
 	slaResults = [],
 	taskNames = [],
 }) {
-	const SLAs = {open: [], resolved: []};
+	const SLAs = {notStarted: [], open: [], resolved: []};
 
 	slaResults.forEach((result) => {
-		SLAs[result.status === 'STOPPED' ? 'resolved' : 'open'].push(result);
+		let slaGroup = '';
+
+		switch (result.status) {
+			case 'STOPPED': {
```

Comment:
@seltong the case statement must be in alphabetical order as well :). Please, fix it.